project/core/templates/about.html

```diff
@@ -185,17 +186,12 @@ <h3 class="title">{% trans "Sponsors" %} </h3>
     </div>
     </main>
 </script>
-{% endblock backbone_template %}
 
-{% block content %}
 {% include "static_nav.html" %}
-    <div id="about"></div>
-{% endblock content %}
-
-{% block extra_js %}
-<!-- Backbone Views File -->
-<script src="{% static "js/static/about_view.js" %}" type="text/javascript"></script>
+<div id="about"></div>
+<script src="{% static " js/static/about_view.js" %}" type="text/javascript"></script>
 <script type="text/javascript">
     var aboutView = new cw.AboutView();
 </script>
-{% endblock extra_js %}
+
+{% block content %}
```

Comment:
This should be the end of the content block.
```suggestion
{% endblock content %}
```